index.html

```diff
@@ -29,57 +31,56 @@ <h1 class="w3-display-topmiddle w3-container">Trick</h1>
 
             <!-- Rule page will be hidden. Instead it only show a button which will be on the top RIGHT/LEFT side of the Welcome page -->
             <!-- Player will be CLICK on the the RULE Button to view the rules -->
             <!-- Once Player click button, it will display pop-up a BLOCK page with the RULE DISPLAY -->
-            <div class="w3-container">
-                <button id="show-rule-hover" class="w3-button w3-medium" onclick="showRuleButton()">Rules</button>
-                <div id="game-rule-text" class="w3-border w3-round-large">
+            <div class="game-rule-card">
+                <div id="game-rule-text" class="w3-left-align">
                     <h4>GOAL</h4>
                     <p>
-                        The goal of this game is to trick your friends and get the most points to
-                        win. There are two stages to this game; setup and guessing.
+                        Try and trick your friends with the best wrong answers to earn the most points. 
                     </p>
                     <h4>Stage-1</h4>
                     <p>
-                        In the setup stage, you will be shown a question and you will be able
-                        to provide a believable answer that is <strong>NOT</strong> correct.
-                        You will only have 30 seconds to come up with the answer and submit it.
+                        In <strong>Trick Trivia</strong>, players will be shown a question and then
+                        provide the best <strong>wrong</strong> answer.
                     </p>
                     <h4>Stage-2</h4>
                     <p>
-                        In the guessing stage, you will be given a list of answers. One of them
-                        will be the correct answer, but the rest will be a mix of the ones
-                        provided by your friends and ones provided by the game. You will have to
-                        select the correct answer. If you get the correct answer, you will get
-                        3 points. If you are the first to select the correct answer, you will
-                        get an additional 2 points. If someone else picked the answer you gave
-                        in the setup stage, then you will get one more point for each person
-                        picking your answer.
+                        Next, players will be given a list of answers. One of them will be the 
+                        correct answer, but the rest will be a mix of the <strong>wrong</strong> 
+                        answers players provided. If you've played your chance well, you will have 
+                        <strong>tricked</strong> everyone else into picking your wrong answer. 
+                        You will get: 
+                        <ul>
+                            <li>3 points for the correct answer.</li>
+                            <li>2 points for the first correct answer.</li>
```

Comment:
Could you change this to "2 additional points for being the first to provide the correct answer." Just to clearify that being the first doesn't is a good thing.